t/apicast-policy-upstream.t

```diff
@@ -215,16 +215,12 @@ yay, api backend
 }
 --- upstream
   location /path_in_the_rule {
-     content_by_lua_block {
-       require('luassert').are.equal('GET /path_in_the_rule?user_key=uk&a_param=a_value HTTP/1.1',
-                                     ngx.var.request)
-       ngx.say('yay, api backend');
-     }
+     echo $request;
   }
 --- request
 GET /some_path?user_key=uk&a_param=a_value
 --- response_body
-yay, api backend
+GET /path_in_the_rule/some_path?user_key=uk&a_param=a_value HTTP/1.1
```

Comment:
Technically this is a breaking change as it used to be `GET /path_in_the_rule`.

But I was thinking that the upstream policy needs some improvements anyway. For example, stripping the path that it matched from the URL etc. So we can make general improvements to that policy and make it configurable how it operates on URI paths.